<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>341</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Objects</string>
    </property>
    <widget class="QListView" name="objectsListView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>331</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>210</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>341</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Add object</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>331</width>
       <height>231</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Line</string>
      </attribute>
      <widget class="QLineEdit" name="lineStartX">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineStartY">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineStartZ">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>90</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>170</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEndZ">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEndY">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>170</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEndX">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Start coordinates</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>End coordinates</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveLineButton">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>170</y>
         <width>97</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineName">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Point</string>
      </attribute>
      <widget class="QPushButton" name="savePointButton">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>170</y>
         <width>97</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Point coordinates</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>90</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="pointZ">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="pointX">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="pointY">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="pointName">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Polygon</string>
      </attribute>
      <widget class="QPushButton" name="savePolygonButton">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>170</y>
         <width>97</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Start coordinates</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>90</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="polygonZ">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="polygonX">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="polygonY">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QListView" name="pointsListView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>211</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="addPolygonButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>80</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="polygonName">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="Canhamo" name="widgetCanhamo" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>871</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Controls</string>
    </property>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="moveRightButton">
     <property name="geometry">
      <rect>
       <x>668</x>
       <y>70</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Move Right</string>
     </property>
    </widget>
    <widget class="QPushButton" name="moveDownButton">
     <property name="geometry">
      <rect>
       <x>558</x>
       <y>100</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Move Down</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="moveLeftButton">
     <property name="geometry">
      <rect>
       <x>458</x>
       <y>70</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Move Left</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="windowMaxZ">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="windowMinZ">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Window Max Point</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Window Min Point</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updateWindowButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="windowMinX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="windowMaxY">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="resetWindowButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>110</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="moveUpButton">
     <property name="geometry">
      <rect>
       <x>558</x>
       <y>40</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Move Up</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>558</x>
       <y>80</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Step:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="moveStep">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>70</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="windowMaxX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="windowMinY">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>518</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Factor (%):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="zoomFactor">
     <property name="geometry">
      <rect>
       <x>598</x>
       <y>0</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomOutButton">
     <property name="geometry">
      <rect>
       <x>758</x>
       <y>0</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom Out</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomInButton">
     <property name="geometry">
      <rect>
       <x>368</x>
       <y>0</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom In</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Canhamo</class>
   <extends>QWidget</extends>
   <header>forms/canhamo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
